Dockerfile

```
@@ -4,9 +4,31 @@
 # SPDX-License-Identifier: Apache-2.0
 
 # BUILDPLATFORM is an automatic platform ARG enabled by Docker BuildKit.
-# Represents the plataform where the build is happening, do not mix with
+# Represents the platform where the build is happening, do not mix with
 # TARGETARCH
+
+# Stage 1: install dnsperf and its dependencies
+FROM docker.io/library/alpine:3.18.2@sha256:82d1e9d7ed48a7523bdebc18cf6290bdb97b82302a8a9c27d4fe885949ea94d1 as dnsperf-builder
+
+ARG DNSPERF_VERSION=2.13.0
+
+RUN apk update \
+    && apk add curl tar \
+    && apk add ldns-dev ck-dev build-base gcc libtool nghttp2-dev libssl3 libcrypto3
```

```suggestion
    && apk add build-base gcc libtool ldns-dev ck-dev nghttp2-dev libssl3-dev libcrypto3-dev
```

I think the -dev dependencies were probably pulled in thanks to `ldns`, but better to be explicit here as well

---

```
#0 1.680 ERROR: unable to select packages:
#0 1.680   libcrypto3-dev (no such package):
#0 1.680     required by: world[libcrypto3-dev]
#0 1.680   libssl3-dev (no such package):
#0 1.680     required by: world[libssl3-dev]
```

---

Okay, thx!